<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkPropertiesPreferencePage</class>
 <widget class="QWidget" name="QmitkPropertiesPreferencePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="aliasesGroupBox">
     <property name="title">
      <string>Aliases</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="showAliasesCheckBox">
        <property name="text">
         <string>Replace genuine property names by aliases</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="descriptionsGroupBox">
     <property name="title">
      <string>Descriptions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showDescriptionsCheckBox">
        <property name="text">
         <string>Show descriptions</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showAliasesInDescriptionCheckBox">
        <property name="styleSheet">
         <string notr="true">margin-left: 18px;</string>
        </property>
        <property name="text">
         <string>Show aliases of genuine property names</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filtersGroupBox">
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="filterPropertiesCheckBox">
        <property name="text">
         <string>Only show filtered properties</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>204</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showAliasesCheckBox</tabstop>
  <tabstop>showDescriptionsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
